app/components/FormLine.js

```diff
@@ -23,9 +20,33 @@ const FormLine = ({label, value, valueUpdateCB}) => {
         valueUpdateCB(label, value);
     };
 
+    const inputStyle = StyleSheet.create({
+        inputContainer: {
+            position:           'relative',
+            display:            'flex',
+            flexDirection:      'row',
+            backgroundColor:    'white',
+            width:              '70%',
+            marginBottom:       10,
+            paddingTop:         6,
+            paddingBottom:      6,
+            paddingLeft:        '3.333%',
+            paddingRight:       '3.333%',
+        },
+
+        textContainer:
+        {
```

Comment:
styling. Would fix this later with prettier or tslint

Reply:
It's just the basic style for now.